### PR TITLE
ON-20788 # Added date formats to localisationService

### PR DESCRIPTION
**Pull request by Zaxist**

### CODE REVIEW

File: src/localisation-service.ts [hunk 1]
```diff
@@ -15,6 +15,34 @@ export class LocalisationService {
...
+  get longDateFormat() {
+    switch (this.locale) {
+      case 'en-US':
+        return 'MMM Do, YYYY'
+      case 'en-AU':
+      default:
+        return 'Do MMM, YYYY'
+    }
+  }
+  get longDateTimeFormat() {
```
Comment (mymattcarroll) [inline src/localisation-service.ts:36]: can these be renamed to have the library they are for.

Comment (Zaxist) [reply src/localisation-service.ts:36]: Pretty sure its universal. Works for moment, mui and looks like will work for date-fns too

Comment (Zaxist) [reply src/localisation-service.ts:36]: My mistake, its slightly different for date-fns. mui must be using the formatter for whichever lib you use with it

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - `formStoreService`
+- `shortDateFormat`, `longDateFormat` and `longDateTimeFormat` to localisationService
```
Comment (mymattcarroll) [inline CHANGELOG.md:13]:
```suggestion
- `dateFnsFormats` to localisationService
```

**Pull request merged**